HelloWorld.java

```diff
@@ -0,0 +1,21 @@
+
+package com.example.demo;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@SpringBootApplication
+@RestController
+public class Demo1Application {
+  
+    
+    public static void main(String[] args) {
+    SpringApplication.run(Demo1Application.class, args);
+    }
+    
+    @GetMapping("/hello")
+    public String hello() {
+    return "Hello World!";
+    }
```

Comment:
こちらもインデントしてください！
```suggestion
    @GetMapping("/hello")
    public String hello() {
        return "Hello World!";
    }
```

IDEにファイル保存時にフォーマットするように設定を入れましょう！
eclipse save actions
eclipse format
などで調べてみてください！